templates/static/js/terminal_fill.js

```diff
@@ -33,7 +33,7 @@ function terminal(el)
             return 0;\n\
         }';
     } else if (value == 2) {
-        textarea.value = 'using System;\n\
+        textarea.value = textarea.value = 'using System;\n\
```

Comment:
`textarea.value` is repeated here. Please rectify.

Reply:
Sorry about that. Fixed and also fixed the spacing in a comment.